docs/release_docs/Pr-ci-cd_ReleaseFlow.md

```diff
@@ -0,0 +1,26 @@
+## PR-CI-CD
+ 
+ PR-CI-CD can be trigger by creating a pull request from ``` Feature/* ``` branch to merge into develop branch.
+ ![PR-ci-cd_workflow](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/pr-ci-cd_workflow.png?raw=true)
+ ![pr](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Pr-ci-cd_example.png?raw=true)
+ 
+### JOBS
+
+ These are the jobs used in PR-ci-cd workflow
+ 
+ #### CodeScan-ESLint
+  In this job we are analyzes the code developer wrote against some rules for stylistic or programmatic errors.
+ 
+  * This job use static code analysis tool which identify problematic patterns found in application source code.
+  * This job uses [Super-linter](https://github.com/marketplace/actions/super-linter) action to run this job. 
+ 
+ #### Build-Docker-Image
+ In this job we are validating whether ECR image building properly or not.
```

Comment:
In this job, we validate the Dockerfile and test the image build process to identify any issues incurred in the build process to recent code changes.